src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+@import "../utils/variables";
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+  box-sizing: border-box;
+  // margin: 51px 129px 0;
+  margin: 0 auto;
+  max-width: (4 * $card-width) + (4 * $card-between-hor);
+  padding: 50px 0;
+
+  &__card {
+    width: $card-width;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    margin-right: $card-between-hor;
+    margin-bottom: $card-between-ver;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-top: 0;
+    }
+
+    .card {
+      &__image img {
+        // width: 162px;
+        height: 134px;
+        margin: 32px 20px 0;
+      }
+
+      &__title {
+        font-size: 12px;
+        line-height: 18px;
+        font-weight: 500;
+        color: $high-blue-color;
+        margin: 36px 16px 0;
+      }
+
+      &__text {
+        font-size: 10px;
+        font-weight: 400;
+        color: $grey-color;
+        margin: 4px 84px 16px 16px;
```

Comment:
too much margins.
consider an option to apply paddings on `.card`: `padding: 32px 16px 16px`
and get rid of all horizontal paddings of children elements

Reply:
then `display: flex` can help you with justifying and aligning